### PR TITLE
Changed the option to enable the smart threading with the option to disable this

### PR DESCRIPTION
**Pull request by annando**
The config value is now reversed. Now you have to actively disable the smart threading. The smart threading showed it value for some time now - without having obvious flaws. So it should be the default action now. But for those who dislike it, there still is the possibility to disable it.

### CODE REVIEW

File: include/conversation.php [hunk 1]
```diff
@@ -1479,7 +1479,7 @@ function conv_sort(array $item_list, $order)
 		$parents[$i]['children'] = sort_item_children($parents[$i]['children']);
 	}
 
-	if (PConfig::get(local_user(), 'system', 'smart_threading', 0)) {
+	if (!PConfig::get(local_user(), 'system', 'no_smart_threading', 0)) {
```
Comment (MrPetovan) [inline include/conversation.php]: Please rename to `dumb_threading`.

**Comment by AlfredSK** [comment]
Seems like smart-threading doesn't work on community pages now.
Situation:
Network page -> smart threading
Community page -> dumb threading

**Comment by MrPetovan** [comment]
Did it ever work?

**Comment by AlfredSK** [comment]
Hmm. I'm not sure. :-) I just saw one thread on global community page that doesn't look like smart threading. Some comments only used ~50% of the available width.

**Pull request merged**